apps/portals/ampals/src/config/routesConfig.tsx

```diff
@@ -7,7 +7,17 @@ import { MarkdownSynapse } from 'synapse-react-client'
 import HomePage from '../pages/HomePage'
 import explorePageRoutes from './explorePageRoutes'
 import ExploreWrapper from '@sage-bionetworks/synapse-portal-framework/components/Explore/ExploreWrapper'
+import AMPALSResearchPageLayout from '@sage-bionetworks/synapse-portal-framework/components/ampals/AMPALSResearchPageLayout'
 import { searchPageChildRoutes } from '@/pages/AMPALSSearchPage'
+import { Typography, Link } from '@mui/material'
+//import header images for Resources pages
+import ResearcherResourcesHeaderSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/researcher_resources_slat.png'
+import ContributorResourcesHeaderSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/data_contributor_resources_slat.png'
+import PWLEResourcesHeaderSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/pwles_resources_slat.png'
+import PublishingRequirementsSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/publishing_requirements.png'
+import ContributeDataSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/contribute_data_slat.png'
```

Comment:
suggestion: Since these are not React components, use camelCase
```suggestion
import researcherResourcesHeaderSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/researcher_resources_slat.png'
import contributorResourcesHeaderSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/data_contributor_resources_slat.png'
import pwleResourcesHeaderSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/pwles_resources_slat.png'
import publishingRequirementsSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/publishing_requirements.png'
import contributeDataSlat from '@sage-bionetworks/synapse-portal-framework/components/ampals/assets/contribute_data_slat.png'
```

Also, if some of these are AMP-ALS specific, consider moving those to `portals/ampals/src/assets` (might need to create this folder)

Reply:
@nickgros how do you make the determination to keep things under `portals/` vs within `synapse-portal-framework/`? I thought I was following the norm for the repo based on what I saw when working on some [Elite specific components](https://github.com/Sage-Bionetworks/synapse-web-monorepo/tree/main/apps/synapse-portal-framework/src/components/eliteportal/assets) previously.

Reply:
Thanks this is really helpful. I'll move these assets to `/portals/`